ext/MitsubaLoader/CMitsubaMetadata.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __IRR_C_MITSUBA_METADATA_H_INCLUDED__
+#define __IRR_C_MITSUBA_METADATA_H_INCLUDED__
+
+#include "irr/asset/IAssetMetadata.h"
+#include "../../ext/MitsubaLoader/CGlobalMitsubaMetadata.h"
+
+namespace irr
+{
+	namespace ext
+	{
+		namespace MitsubaLoader
+		{
+			//! A class to derive mitsuba mesh loader metadata objects from
+
+			class CMitsubaMetadata : public asset::IAssetMetadata
+			{
+				public:
+
+					CMitsubaMetadata(core::smart_refctd_ptr<CGlobalMitsubaMetadata> _mitsubaMetadata) : mitsubaMetadata(std::move(_mitsubaMetadata)) {}
+
+					std::string getName() const
+					{
+						return name;
```

Comment:
what's the `name` used for?

Reply:
Nothing apparently, and it's not even initialized, It's used only in 
```
	if (_params.loaderFlags & IAssetLoader::ELPF_LOAD_METADATA_ONLY)
	{
		auto emptyMesh = core::make_smart_refctd_ptr<asset::CCPUMesh>();
		m_manager->setAssetMetadata(emptyMesh.get(), core::make_smart_refctd_ptr<ext::MitsubaLoader::CMitsubaMetadata>(parserManager.m_globalMetadata));
		return SAssetBundle({ std::move(emptyMesh) });
	}
```
i think it's Arek's code

Reply:
ok then get rid of the `name`

Arek's code is a fun way to extract camera transforms without loading entire scenes (for the denoiser)